applications/ray/kuberay-tpu-webhook/certs/cert.yaml

```diff
@@ -2,15 +2,15 @@ apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
   name: selfsigned-issuer
-  namespace: default
+  namespace: ray-system
```

Comment:
I like `ray-system` as a namespace, but I also think it should match the default namespace used in our Ray templates: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/ray/variables.tf#L42

Reply:
@imreddy13 can we change the default namespace for Ray to be `ray-system` or will that require other changes for RAG?

Reply:
Resolving this for now, but we can follow up on making the default ray namespace uniform across the repo.